packages/core/src/fhir-to-cda/cda-templates/components/__tests__/medications.test.ts

```diff
@@ -62,11 +63,8 @@ describe("buildMedications", () => {
       pastDateCda,
       pastDateHumanReadable,
     };
-    // TODO: Remove the console.log after we fix the tsconfig to ignore "unused" vars,
-    // since `eval()` isn't explicitly using them
-    console.log("params", params);
-
-    const xmlContent = eval("`" + getXmlContentFromFile(filePath) + "`");
+    const xmlTemplate = _.template(getXmlContentFromFile(filePath));
+    const xmlContent = xmlTemplate(params);
```

Comment:
We name functions to represent an action, so maybe instead of `xmlTemplate` we can call it `interpolateTemplate` or `applyToTemplate`?

I called it `interpolate` on E2E tests (`checkConsolidated()`) - now I see that it misses "template" of maybe could be simpler like the "apply to template" above... 😅

Reply:
I like `applyToTemplate`